client/src/components/Photo.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Photo = (props) => {
+  let photoUrl = props.photoUrl ? props.photoUrl.url : 'https://i.ytimg.com/vi/QggJzZdIYPI/maxresdefault.jpg';
```

Comment:
Make this string a const (variable name all capital letters at the top of the file).

Reply:
Name suggestion: `DEFAULT_PHOTO_URL`